### PR TITLE
Table refactor and support continuous update

### PR DESCRIPTION
**Pull request by ngjaying**
#755

### CODE REVIEW

**Comment by jinfahua** [comment]
Could you add an end-2-end SVT case?

File: docs/en_US/sqls/tables.md [hunk 1]
```diff
@@ -13,40 +15,40 @@ CREATE TABLE
...
-  fileType: json
-  # The directory of the file relative to kuiper root or an absolute path.
-  # Do not include the file name here. The file name should be defined in the stream data source
-  path: data
+		id BIGINT,
+		name STRING
+	) WITH (DATASOURCE="lookup.json", FORMAT="JSON", TYPE="file");
+
+SELECT * FROM demo INNER JOIN table1 on demo.id = table1.id
 ```
```
Comment (jinfahua) [inline docs/en_US/sqls/tables.md:37]: Add a sample file content for this `table` definition.

**Review by jinfahua — CHANGES_REQUESTED** [review]
A sample file content...

**Comment by ngjaying** [comment]
> 1. Where is the `lookup.json`?
Already in the fvt_scripts. It was created before as an input of unit tests.

**Pull request merged**